packages/mask/src/extension/popups/pages/Personas/components/PersonaList/index.tsx

```diff
@@ -42,49 +52,148 @@ const useStyles = makeStyles()({
     },
     trashIcon: {
         fontSize: 12,
-        stroke: '#1C68F3',
         marginLeft: 6,
         cursor: 'pointer',
+        width: 12,
+        height: 12,
+    },
+    tick: {
+        fill: 'none',
+        fontSize: 18,
+    },
+    edit: {
+        fontSize: 16,
+        stroke: '#1C68F3',
+        fill: 'none',
+        marginLeft: 10,
+        cursor: 'pointer',
+    },
+    copy: {
+        fontSize: 12,
+        fill: '#1C68F3',
+        marginLeft: 4,
+        cursor: 'pointer',
+    },
+    setting: {
+        fontSize: 12,
+        cursor: 'pointer',
+        fill: 'none',
+        stroke: '#1C68F3',
+        marginLeft: 4,
     },
 })
 
+export const PersonaList = memo(() => {
+    const { personas, setSelectedPersona, currentPersona } = PersonaContext.useContainer()
+    const navigate = useNavigate()
+
+    const onLogout = useCallback(
+        (persona: PersonaInformation) => {
+            setSelectedPersona(persona)
+            navigate(PopupRoutes.Logout)
+        },
+        [setSelectedPersona],
+    )
+
+    const onChangeCurrentPersonas = useCallback((identifier: ECKeyIdentifier) => {
+        Services.Settings.setCurrentPersonaIdentifier(identifier).then(() => navigate(PopupRoutes.Personas))
```

Comment:
Is it should return a `Promise` value?

Reply:
I don't think this needs to return anything. Just a change in behavior